modules/json/src/main/scala/gem/json.scala

```diff
@@ -36,11 +37,19 @@ package object json {
     casecodec2(Observation.Id.apply, Observation.Id.unapply)("program-id", "index")
 
   // OffsetP maps to a signed angle in arcseconds
-  implicit val OffsetPMeta: CodecJson[OffsetP] =
+  implicit val OffsetPCodec: CodecJson[OffsetP] =
```

Comment:
It can't do it for some reason, I think because `OffsetP` and `OffsetQ` extend `AnyVal`.